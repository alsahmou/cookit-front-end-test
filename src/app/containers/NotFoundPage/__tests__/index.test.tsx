import * as React from 'react';
import { NotFoundPage } from '..';
import { createRenderer } from 'react-test-renderer/shallow';

const shallowRenderer = createRenderer();

describe('<NotFoundPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<NotFoundPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
