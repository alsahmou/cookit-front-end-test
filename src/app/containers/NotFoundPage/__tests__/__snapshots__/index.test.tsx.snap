// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      404 Page Not Found
    </title>
    <meta
      content="Page not found"
      name="description"
    />
  </Helmet>
  <NavBar />
  <NotFoundPage__Wrapper>
    <NotFoundPage__Title>
      4
      <span
        aria-label="Crying Face"
        role="img"
      >
        ðŸ˜¢
      </span>
      4
    </NotFoundPage__Title>
    <P>
      Page not found.
    </P>
    <Link
      to="/"
    >
      Return to Home Page
    </Link>
  </NotFoundPage__Wrapper>
</React.Fragment>
`;
